be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -111,6 +111,19 @@ class OptionalStoredColumnReader : public StoredColumnReader {
         *num_levels = _levels_parsed;
     }
 
+    void append_default_levels(size_t row_nums) {
+        if (_need_parse_levels) {
```

Comment:
what does this field mean?

Reply:
Only subfield of struct need parse_levels, otherwise we don't need return levels for optional column.